MAUI/ProgressBar/getting-started.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Getting started with .NET MAUI ProgressBar control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI ProgressBar (Progress Bar) control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Getting started with .NET MAUI ProgressBar (Progress Bar)
+
+This section explains the steps required to add the progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill and track fill. This section covers only basic features needed to know to get started with Syncfusion progress bar.
+
+## Creating an application using the .NET MAUI ProgressBar
+
+* Create a new .NET MAUI application in the Visual Studio.
+
+* Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add ProgressBar to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.ProgressBar] then install that.
+
+### Register the handler
+
+Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# tabtitle="~/MauiProgram.cs" hl_lines="17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace MyProject
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+
+## Adding namespace
+
+Add the following namespace.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar"
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.ProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Initializing progress bar
+
+Create an instance for the progress bar control, and add it as content. The progress bar control has two variants: SfLinearProgressBar and SfCircularProgressBar. Each renders the progress in its own shape, such as rectangle and circle, respectively. Initialize both the progress bars with a progress value using the Progress property as demonstrated in the following code sample.
```

Comment:
Create an instance for the progress bar control, and add it as content. The progress bar control has two variants: The SfLinearProgressBar and SfCircularProgressBar. Each renders the progress in its own shape, such as a rectangle and circle, respectively. Initialize both progress bars with a progress value using the Progress property as demonstrated in the following code sample.

Reply:
done